source/_twig-components/functions/faker.function.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$function = new Twig_SimpleFunction('faker', function ($string, $options="") {
+  $faker = Faker\Factory::create();
+  if ($options != "") {
+    return $faker->$string($options);
```

Comment:
Relying on options to indicate whether $string is a callable is dicey. Is there a reason to not instead or additionally use http://php.net/is_callable?